### DISCUSSION TITLE
Docs on awsscv.sf?

### DISCUSSION THREAD

**Discussion by gorgonzoleo** [amazon-connect/amazon-connect-salesforce-scv#79] [Q&A] [answered]
Hi there, is there any documentation on the package awsscv.sf? I assumed that it was simple_salesforce, but seems to be slightly different. Particularly, I'd like to see what's the syntax for updating a record.

**Comment by gorgonzoleo** [comment, marked as answer]
Figured that the update is sf.update(objectName, recordId, dataDict)
objectName = string with the SObject API Name
recordId = string with the Id of the record to be updated
dataDict = dictionary with fields and values to be updated
But it would be helpful to have more docs about awsscv.sf.

**Reply by dougjaso** [reply]
There is no documentaiton for it as it is not intended to be used as a standalone API function. All SCV deployments also have an InvokeSalesforceRestApiFunction included that should be used for normal circumstances. This version is included in our common layer to simplify the code examples.
